odoo/models.py

```diff
@@ -5521,6 +5521,33 @@ def onchange(self, values, field_name, field_onchange):
 
         return result
 
+    #
+    # Cleanup
+    #
+
+    @api.model
+    def _expunge_bad_records(self):
+        """
+        Expunge bad records from the current model.
```

Comment:
I would write instead: "Clean up the data from the current model after some module deinstallation."